### PR TITLE
Issue #8, add tests for ChildrenElement class

### PR DESCRIPTION
**Pull request by andrii-z4i**
- ChildrenElement;
- split_hyperlink;

### CODE REVIEW

File: xmind/tests/test_sheet_element.py [hunk 1]
```diff
@@ -5,7 +5,6 @@
 from xmind.core.const import *
```
Comment (andrii-z4i) [inline xmind/tests/test_sheet_element.py:5]: @mykola-pysarev please be aware that there are no changes in this file except reformatting

File: xmind/tests/test_topic_children_element.py [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+        self.assertEqual(3, self._workbook_mixin_element_mock.call_count)
+        self.assertListEqual(
+            [
+                call(_element1, 'a1', None),
+                call(_element2, None, 'b2'),
+                call(_element3, 'a3', 'b3')
+            ],
+            self._workbook_mixin_element_mock.call_args_list
+        )
+
```
Comment (vlad-shulika) [inline xmind/tests/test_topic_children_element.py]: Forgot about __self.assertEqual(TAG_CHILDREN, _element.TAG_NAME)__

Comment (vlad-shulika) [reply xmind/tests/test_topic_children_element.py]: and below in another tests

**Review by mykola-pysarev — COMMENTED** [review]
Everything looks good, except Vlad's comment!

**Comment by andrii-z4i** [comment]
@vlad-shulika , @mykola-pysarev comments are addressed

**Review by vlad-shulika — APPROVED** [review]

**Pull request merged**